malariagen_data/ag3.py

```diff
@@ -341,3 +347,100 @@ def _results_cache_add_analysis_params(self, params):
         super()._results_cache_add_analysis_params(params)
         # override parent class to add AIM analysis
         params["aim_analysis"] = self._aim_analysis
+
+    def load_inversion_tags(self, inversion: str):
+        # needs to be modified depending on where we are hosting
+        path = "malariagen_data/resources/karyotype_tag_snps.csv"
+        df_tag_snps = pd.read_csv(path, sep=",")
```

Comment:
```suggestion
        import importlib.resources
        from . import resources
        with importlib.resources.path(resources, "karyotype_tag_snps.csv") as path:
            df_tag_snps = pd.read_csv(path, sep=",")
```

I think this is the Pythonic way to access a file that is included in the package.

Also you'll probably need to add a `MANIFEST.in` file at the root of the repository which has the line:

```
include malariagen_data/resources/*
```

...to make sure the files will get included when the package is packaged.

Reply:
Thanks, this is good to know. Currently AnoExpress reads the files it needs from the github repo urls - this is much better!!